recipes/icon/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set name = "icon" %}
+{% set version = "9.5.2" %}
+{% set build = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://zenodo.org/record/5213700/files/eschen42/portableIcon-v{{ version }}_cf_{{ build }}_rev2.zip  # [win]
+  sha256: 325f34453e62c49e4ceeb1580ee303cc72845230dda0f558d630bfe2bf1ac5e2  # [win]
+  url: https://github.com/gtownsend/icon/archive/refs/heads/master.zip  # [linux64]
+  sha256: 6c118b2dcc26b6ea3c4129cc82d110a275d3c806749211773fe34c106949efb0  # [linux64]
+
+build:
+  number: {{ build }}
+  skip: True  # [osx]
+
+# attempting to address mysterious Centos and Linux build failures
+requirements:  # [linux64]
+  build:  # [linux64]
+    - {{ compiler('c') }}  # [linux64]
```

Instead of binary repackaging, can you try building with MinGW on Windows?
```suggestion
    - {{ compiler('c') }}  # [linux64]
    - {{ compiler('m2w64') }}  # [win]
```

---

> 
> 
> Instead of binary repackaging, can you try building with MinGW on Windows?

@xhochy
I will but doubt the prospects for tolerably quick success.
If necessary, I will skip the Windows build for now.
Does that sound like an acceptable solution?

---

If you don't have enormous  pressure, do here. In staged-recipes, you will more easily get help. In the most likely case, a `bld.bat` with `bash -lc build.sh` could be sufficient.

---

No, I think that that won't work, actually, because there are a lot of POSIX headers (like grp.h) to include that only Cygwin provides.